src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const headers = [...document.querySelector('thead').children[0].children];
+const tbody = document.querySelector('tbody');
+const peopleRows = [...document.querySelector('tbody').children];
```

Comment:
Instead of using 'children', use 'querySelectorAll' to get all the 'tr' elements from 'tbody'.